build/common.targets

```
@@ -1,5 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0">
+
+  <!-- Set AssemblyVersion in targets as it is TFM dependent. -->
+  <PropertyGroup Condition="'$(AssemblyVersion)' == '' and '$(SemanticVersion)' != ''">
```

When is `SemanticVersion` empty at this point?
I'm not sure why we'd need that check.

---

Yes, the condition isn't necessary:

```suggestion
  <PropertyGroup Condition="'$(AssemblyVersion)' == ''">
```